tests/integration.spec.coffee

```diff
@@ -125,7 +125,7 @@ describe 'SDK Integration Tests', ->
 				mockResin = getSdk(opts)
 				m.chai.expect(mockResin).to.include.keys(validKeys)
 
-	describe 'given a not logged in user', ->
+	describe.skip 'given a not logged in user', ->
```

Comment:
It's probably easier in general to just put `.only` on each of the sets of tests you do want to run, rather than excluding every other batch.